docs/en/expressions.md

```diff
@@ -0,0 +1,28 @@
+# Expressions
+
+To evaluate JavaScript code inside the HTML use curly brackets `{}`.
+Curly brackets can handle most of any valid JavaScript expression.
+
+```html
+<div>{'Hello ' + 'World!'}</div>
+```
+
+With curly brackets you can also inject variables into the template.
+
+```html
+<div>Hello {name}!</div>
+```
+
+Inside the expression you can use built-in filters. Filters are responsible for easy mapping value to another form.
```

Comment:
`The expression can be passed into a filter. Filters are responsible for transforming the value. They're simply a predefined functions that you can use. For more information, visit pure-utilities.`